public/js/poll/view.js

```diff
@@ -86,9 +86,11 @@
 			poll = View.parseResults(poll);
 			for (var i = 0, l = poll.options.length; i < l; i++) {
 				var option = poll.options[i],
-					optionView = pollView.find('#pollResult' + option.id);
+				    optionView = pollView.find('.poll-view-result[data-poll-result="'+option.id+'"]');
 				optionView.find('.poll-view-result-percentage').text(option.percentage + '%');
 				optionView.find('.poll-view-result-progressbar').css('width', option.percentage + '%');
+                		optionView.find('.poll-view-result-votecount').text(option.votecount + ' votes');
+
```

Comment:
For some reason this wasn't even being updated at all. The others weren't updated because the ```optionView``` wasn't being set right above but vote count never got any love...